mobile/lib/common/settings/channel_screen.dart

```diff
@@ -71,60 +89,244 @@ class _ChannelScreenState extends State<ChannelScreen> {
-                          type: ValueType.text,
-                          value: channelStatus,
-                          label: "Channel status",
-                          labelTextStyle: const TextStyle(fontSize: 18),
-                          valueTextStyle:
-                              const TextStyle(fontWeight: FontWeight.bold, fontSize: 18),
-                        ),
-                      ],
-                    )),
-                Visibility(
-                    visible: channelStatusNotifier.isClosing(),
-                    child: Padding(
-                        padding: const EdgeInsets.only(top: 20, left: 10, right: 10),
-                        child: RichText(
-                            text: const TextSpan(
-                                style: TextStyle(color: Colors.black, fontSize: 18),
-                                children: [
-                              TextSpan(
-                                  text: "Your channel with 10101 is being closed on-chain!\n\n",
-                                  style: TextStyle(fontWeight: FontWeight.bold)),
-                              TextSpan(
-                                  text:
-                                      "Your off-chain funds will return back to your on-chain wallet after some time.\n\n"),
-                              TextSpan(
-                                  text:
-                                      "If you had a position open your payout will arrive in your on-chain wallet soon after the expiry time. \n")
-                            ]))))
+                Expanded(
+                  child: SingleChildScrollView(
+                      child: Column(
+                    children: [
+                      Padding(
+                          padding: const EdgeInsets.only(
+                              top: 10.0, bottom: 20.0, left: 14.0, right: 14.0),
+                          child: Column(
+                            children: [
+                              const SizedBox(height: 20),
+                              ValueDataRow(
+                                type: ValueType.text,
+                                value: channelStatus,
+                                label: "Channel status",
+                                labelTextStyle: const TextStyle(fontSize: 18),
+                                valueTextStyle:
+                                    const TextStyle(fontWeight: FontWeight.bold, fontSize: 18),
+                              ),
+                            ],
+                          )),
+                      const Divider(height: 1, thickness: 1, indent: 5, endIndent: 5),
+                      Column(
+                        children: <Widget>[
+                          const SizedBox(height: 10.0),
+                          Theme(
+                            data: Theme.of(context).copyWith(dividerColor: Colors.transparent),
+                            child: ExpansionTile(
```

Comment:
Most of those `ExpansionTile` look the same with only being the title and the list of channels being different. 
Would you mind extracting this into a function or a widget to remove duplication please?